notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/OpprettOppgaveLoeser.kt

```diff
@@ -26,17 +34,34 @@ class OpprettOppgaveLoeser(
         }
 
     override fun onBehov(behov: Behov) {
+        val utloesendeMelding = behov.jsonMessage.toJson().parseJson()
         val forespoerselId = behov.forespoerselId
+        val transaksjonId = utloesendeMelding.toMap()[Key.UUID]
+            ?.fromJson(UuidSerializer)
+            .let {
```

Comment:
hva med: 
`        val transaksjonId = utloesendeMelding.toMap()[Key.UUID]
            ?.fromJson(UuidSerializer)
            ?: UUID.randomUUID() 
            .also {  
                sikkerLogger.error(
                    "Mangler transaksjonId i ${simpleName()}. Erstatter med ny, tilfeldig UUID '$it'.\n${utloesendeMelding.toPretty()}"
                )  
            }`